src/browser/styles/events.scss

```diff
@@ -0,0 +1,90 @@
+@import 'variables';
+@import 'mixins';
+
+$list-font-size : 1em;
+$event-highlight-color: #FFA100;
+$event-description-color: #9E9E9E;
+
+/****** Events ******/
+
+.event-view-container {
```

Comment:
You'll see a lot of em's here. That's because I'd like everything to resize according to the font. If not, it's best to do rem or pixels.